src/katgpucbf/xbgpu/engine.py

```diff
@@ -420,10 +423,19 @@ def __init__(
             send_enabled=init_send_enabled,
         )
 
-        self._populate_sensors()
+        self._populate_sensors(seed)
 
-    def _populate_sensors(self) -> None:
+    def _populate_sensors(self, seed: int) -> None:
         sensors = self.engine.sensors
+        sensors.add(
+            aiokatcp.Sensor(
+                str,
+                f"{self.name}.dither-seed",
```

Comment:
I don't think we have any other sensors with "bpipeline" in the name - we generally report things against individual beams, even when the value is the same for all the beams e.g. chan-range.

Reply:
Yeah I did feel a bit uncomfortable adding it this way, but good point Re: chan-range sensor. Updating accordingly.